testSetup.ts

```diff
@@ -1,3 +1,4 @@
+import "core-js/features/array/flat-map";
```

Comment:
This makes our modules no longer `pure`, right?

Reply:
re: #24

Reply:
This is just for test setup and isn't included for the actual build. But you do raise an important point. I know that we are using ES2015+ `Array.prototype` methods like `.every` and `.includes`, but perhaps we should be more specific about BYO polyfills?

Reply:
good point. Can you raise a quick Issue so we can talk about it sometime